src/v3/src/components/WebAuthNSubmitButton/WebAuthNSubmitButton.test.tsx

```diff
@@ -76,15 +76,14 @@ describe('WebAuthNControlSubmitControl Tests', () => {
     });
   });
 
-  it('should render loading spinner when there is a pending request', async () => {
+  it('should render disabled button when there is a pending request', async () => {
     mockLoading.mockReturnValue(true);
     const { queryByTestId } = render(<WebAuthNSubmitButton {...props} />);
 
-    const button = queryByTestId('button');
-    const spinner = queryByTestId('okta-spinner');
+    const button = queryByTestId('button') as HTMLButtonElement;
 
-    expect(button).toBeNull();
-    expect(spinner).toBeInTheDocument();
+    expect(button).toBeInTheDocument();
+    expect(button.getAttribute('disabled')).not.toBeUndefined();
```

Comment:
Does `.toBe(true)` work here or is the attribute only "not `undefined`"?

Reply:
Yea the `disabled` attribute just gets added to the element without a value, so like 

```
<button ... disabled>Some Label</button> 
```

So when I try to check the value it is an empty string. That is why I just try to get the attr and ensure that the result of the `getAttribute('disabled')` function does not return `undefined` as a way to validate that the property does exist on the element.

Reply:
Got it. I guess I was hoping the library would be a bit smarter and turn it into a boolean since that's really how these  attributes imply